### ISSUE TITLE
Add to Developer Resources Page - Job Hunt Resources Section

### ISSUE THREAD

**Issue by ethanfreire** [Virtual-Coffee/virtualcoffee.io#1098]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of Change

New section on existing page

### URL of existing page

https://virtualcoffee.io/resources

### Context for content change

I noticed that there is a [GitHub discussion](https://github.com/orgs/Virtual-Coffee/discussions/894) that has resources already listed in the Job Search Board Google doc in the virtual coffee Google Drive.

### Proposed solution

 To reduce duplication of the same information related to job search sites. I propose adding the "Job Hunt Resources" section to the "Developer Resources" page. The "Job Hunt Resources" page will have a subpage titled "Job Search Board."

### Resources that can help

The contents of the "Job Search Board" page will include information from both the [GitHub discussion](https://github.com/orgs/Virtual-Coffee/discussions/894) and [Google Doc job search board](https://docs.google.com/document/d/1IssYHG3qVl-CZCxpeTvJ1-JZhxwYGSJxGi7ZEAEGyW8/edit?usp=sharing)

### Collaborators

@juliaseid @BekahHW @danieltott @ayu

### Code of Conduct

- [X] I've read the Code of Conduct and understand my responsibilities as a member of the Virtual Coffee community

**Comment by BekahHW** [comment]
@ethanfreire my biggest concern here is with updating. There's friction to add and remove things from the site. But to add to a doc or discussion is a little easier. Thoughts?

**Comment by CBID2** [comment]
> @ethanfreire my biggest concern here is with updating. There's friction to add and remove things from the site. But to add to a doc or discussion is a little easier. Thoughts?

I agree with @BekahHW! :) Having it as a doc would be best management-wise. It'll also be easier for people who don't code to contribute.

**Comment by ethanfreire** [comment]
We have the Google doc I shared in the tech interview study group google drive with a massive list of job boards. I can maintain that monthly. We would have to add a link on the VC site directly to the Google doc. What do you think of this approach?

**Comment by CBID2** [comment]
> We have the Google doc I shared in the tech interview study group google drive with a massive list of job boards. I can maintain that monthly. We would have to add a link on the VC site directly to the Google doc. What do you think of this approach?

That's a great idea @ethanfreire! :)

**Comment by BekahHW** [comment]
@ethanfreire if we're adding a link to the site, what will go on the dev resources page?

**Comment by ethanfreire** [comment]
A link titled "Job Board List" will link to the Google doc containing the massive job board list I've been maintaining. But the link will only be view-only. Only certain virtual coffee members will be able to edit the shared doc.
